packages/asana/test/adaptor.test.js

```diff
@@ -220,6 +220,7 @@ describe('Adaptor Test', () => {
           path: `/api/1.0/workspaces/${workspaceGid}/tasks/search`,
           query: {
             text: taskName,
+            resource_subtype: 'default_task',
```

Comment:
What is the reason for this change?

We've defaulted this value, so it's not necessary to set it here.

But we're also not testing the default value in anyway that I can see? Nor are we testing that it can be overridden?

Reply:
I have added unit test for `default_taks` and `milestone` option